docs/CodeTransformation.md

```diff
@@ -128,6 +128,12 @@ As can be seen, only `process` or `processAsync` is mandatory to implement, alth
 
 Note that [ECMAScript module](ECMAScriptModules.md) support is indicated by the passed in `supports*` options. Specifically `supportsDynamicImport: true` means the transformer can return `import()` expressions, which is supported by both ESM and CJS. If `supportsStaticESM: true` it means top level `import` statements are supported and the code will be interpreted as ESM and not CJS. See [Node's docs](https://nodejs.org/api/esm.html#esm_differences_between_es_modules_and_commonjs) for details on the differences.
 
+:::info
```

Comment:
Can we flip this around? maybe as a `tip` and something like "make sure your transformation returns a source map so it's possible to map back from the transformed code to the source code in code coverage and error information" or some such? Inline sourcemaps also work, doesn't have to be an explicit return value (although separate is preferred for performance reasons)

Reply:
Sure thing @SimenB. How does this sound?

> Make sure your `TransformedSource` returns a source map, so it is possible to report line information accurately in code coverage and test errors. Inline source maps also work but are slower.

I actually didn't know about inline source maps so when I was writing my own code transformer I had to separate our maps because I didn't know that option existed so having this kind of tip should be helpful.